src/components/utils/include/utils/file_system.h

```diff
@@ -256,14 +256,6 @@ bool ReadBinaryFile(const std::string& name,
 
 bool ReadFile(const std::string& name, std::string& result);
 
-/**
- * @brief Convert special symbols in system path to percent-encoded
- *
- * @param name path to file
- * @return returns converted path.
- */
-const std::string ConvertPathForURL(const std::string& path);
```

Comment:
Since this is a public function, it will need to be deprecated instead of removed for now.

Reply:
Done in 3f4fe792854b71a882f27362609a544830e081e2